data/human/hails.txt

```diff
@@ -905,6 +905,12 @@ phrase "friendly civilian"
 		"Why is it difficult to navigate through Dubhe? Because there's a thick Haze!"
 		"Why do engineers go to Diphda? Because they like to Tinker!"
 		"What is a robot's least favorite planet? Rust!"
+		"What do you call a spaceship with no engines? A groundship!"
+		"Don't trust stairs. They are always up to something."
+		"How many Hestians does it take to change a lightbulb? Four! An electrician does the work, and the butler, manager and housekeeper supervise."
+		"How many Hestians does it take to change a lightbulb? Zero, they don't use such old technology there!"
+		"How many Hestians does it take to change a lightbulb? Zero, it's all out-sourced to off-worlders."
+		"How many Hestians does it take to change a lightbulb? One, they are very educated and capable people."
```

Comment:
We could conceivably change these to list more than just Hestians if we wanted to.
```
		"How many ${demonym} does it take..."
		...

phrase "demonym"
	word
		"Hestian"
		"Earthling"
		"Venusian"
		"Martian"
		"New Bostonian"
		...
```

Reply:
I think it's nicer to create them specifically per planet (based a bit on the planets lore).

Reply:
I just added your suggestions and some more.